scheduler/src/cook/mesos/fenzo_utils.clj

```diff
@@ -0,0 +1,91 @@
+;;
+;; Copyright (c) Two Sigma Open Source, LLC
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;  http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+(ns cook.mesos.fenzo-utils
+  (:require [clojure.tools.logging :as log]
+            [datomic.api :as d])
+  (import java.util.concurrent.TimeUnit
+          org.apache.mesos.Protos$Offer
+          com.netflix.fenzo.TaskAssignmentResult
+          com.netflix.fenzo.TaskScheduler
+          com.netflix.fenzo.VirtualMachineLease
+          com.netflix.fenzo.plugins.BinPackingFitnessCalculators))
+
+(defn extract-message
```

Comment:
It looks like they added a getter about a month ago:

https://github.com/Netflix/Fenzo/commit/0e7595381d04441cb0adbd7235300e366d816171

Is it worth trying to upgrade Fenzo instead of doing this?